python_agent_utils/messages/connection.py

```diff
@@ -96,31 +100,30 @@ def build(label: str, my_did: str, my_vk: str, endpoint: str) -> Message:
 
         @staticmethod
         def validate(request):
-            validate_message(
+            request.validate(
                 [
-                    ('@type', Connection.Message.REQUEST),
+                    ('@type', Connection.REQUEST),
                     '@id',
                     'label',
-                    'connection'
-                ],
-                request
+                    Connection.CONNECTION
+                ]
             )
 
-            validate_message(
+            Message.validate_message(
                 [
-                    'did',
-                    'did_doc'
+                    DIDDoc.DID,
+                    DIDDoc.DID_DOC
                 ],
-                request['connection']
+                request[Connection.CONNECTION]
             )
 
-            Connection.DIDDoc.validate(request['connection']['did_doc'])
+            DIDDoc.validate(request[Connection.CONNECTION][DIDDoc.DID_DOC])
```

Comment:
`request[Connection.CONNECTION][DIDDoc.DID_DOC]` is not a `Message` so the `did_doc.validate` in the first line of `DIDDoc.validate` is failing with "dict has no attribute 'validate'"

Reply:
Changed. Thanks